src/user-store.js

```diff
@@ -194,6 +194,20 @@ class UserStore {
       })
       .then(() => this.backend.put('users', userKey, user))
   }
+  /**
+   * Permanently deletes the files of a user
+   *
+   * @param user {UserAccount}
+   *
+   * @return {Promise}
+   */
+
+  deleteUser (user) {
+    let userKey = UserStore.normalizeIdKey(user.id)
+
+    return Promise.resolve()
+      .then(() => this.backend.del('users', userKey))
```

Comment:
What's the `Promise.resolve().then` stuff about?
This should just be `[return] this.backend.del('users', userKey)`.

Reply:
Yes, you're right, don't need the promise-stuff; I'll can fix this right away